docs/pages/machine-id/guides/ansible.mdx

```diff
@@ -6,30 +6,29 @@ description: Using Teleport with Ansible
 Ansible uses the OpenSSH client by default. Teleport supports SSH protocol and
 works as SSH jumphost.
 
-In this guide we will configure OpenSSH client to work with Teleport Proxy
-and run a sample ansible playbook.
+In this guide we will configure OpenSSH client to work with Machine ID
+and run a sample Ansible playbook.
 
 ## Prerequisites
 
-- Installed [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= (=teleport.version=)
-- `tsh` client tool  >= (=teleport.version=)
-- `ssh` openssh tool
-- `ansible` >= (=ansible.min_version=)
-- Optional tool `jq` to process `JSON` output.
+* Installed [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= (=teleport.version=)
+* `tbot` client tool  >= 9.0.0
+* `ssh` OpenSSH tool
+* `ansible` >= (=ansible.min_version=)
+* Optional tool `jq` to process `JSON` output
 
-## Step 1/3. Login and configure SSH
+## Step 1/3. Start Machine ID
```

Comment:
I think it would be better not to base a "Step n" section on an external guide, since that means that a reader will need to switch contexts. Some possible alternatives:

- Add something like this as a Prerequisite step: "A host with Machine ID running. See the [Machine ID Getting Started Guide](../../docs/pages/machine-id/getting-started.mdx) for details."
- Copy and paste steps 1-3 of the Machine ID Getting Started Guide into the Ansible guide. Readers who already know how Machine ID works can speed through these sections while readers new to Machine ID won't have to leave the page.
- Since the Ansible-related steps are fairly short, it could even make sense to include them in the Machine ID Getting Started guide after Step 4, and not have this separate guide.

Reply:
@ptgott Updated and rolled a lot of this into `Prerequisites` section. I think we should still have seperate guides as there is a lot of context around the different integrations where the Getting Started guide might get too messy especially as the number of integrations grows.

However, as docs owners, I'll let you and @xinding33 decide.